src/game/clients/CAccount.cpp

```diff
@@ -593,8 +594,9 @@ void CAccount::DeleteChars()
 			CChar * pChar = m_Chars.GetChar(--i).CharFind();
 			if (pChar != nullptr)
 			{
+				pChar->NotifyDelete(1);
```

Comment:
This is not needed, since it's already called by CChar::Delete, and you can pass fForce there